### DISCUSSION TITLE
So uhh

### DISCUSSION THREAD

**Discussion by Squidbeak431** [3kh0/ext-remover#542] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
For some reason when I click the bookmark on webstorex nothing happens, I click it and it doesn't send me to the thing that lets me disables stuff. Does anyone know why? It was working a few days ago. I have version 105 so the version isn't the problem right?

**Comment by funkymokey** [comment]
What bookmark exactly?

**Reply by Squidbeak431** [reply]
It's just a random bookmark, I just bookmark a random place, add the text as the url, click it and it usually works

**Comment by S-PScripts** [comment, marked as answer]
why are these all unmarked bruh
